playground/TestShop/AppHost/Program.cs

```diff
@@ -12,7 +12,10 @@
                             .WithReference(catalogDb)
                             .WithReplicas(2);
 
-var messaging = builder.AddRabbitMQ("messaging").PublishAsContainer();
+var rabbitMqPw = builder.AddParameter("rabbitmqpw", secret: true);
```

Comment:
(nit) `rabbitmqpw` is really hard to read/parse at first.

```suggestion
var rabbitMqPassword = builder.AddParameter("rabbitmq-password", secret: true);
```